lms/djangoapps/grades/migrations/0001_initial.py

```diff
@@ -45,10 +46,6 @@ class Migration(migrations.Migration):
         ),
         migrations.AlterUniqueTogether(
             name='persistentsubsectiongrade',
-            unique_together=set([('user_id', 'usage_key')]),
-        ),
-        migrations.AlterIndexTogether(
-            name='persistentsubsectiongrade',
-            index_together=set([('user_id', 'usage_key')]),
+            unique_together=set([('user_id', 'course_id', 'usage_key')]),
```

Comment:
Consider using set literals `{('user_id', 'course_id', 'usage_key')}` instead of calling the `set()` constructor They are faster (don't require creating a separate list, and making a function call), clearer, (less line noise), and idiomatic python.